plugins/flytekit-envd/flytekitplugins/envd/image_builder.py

```diff
@@ -176,6 +176,46 @@ def build():
         else:
             envd_config += '    io.copy(source="./", target="/root")\n'
 
+    if image_spec.copy:
+
+        def add_envd_copy_command(src_path: pathlib.Path, envd_config: str) -> str:
+            envd_version = metadata.version("envd")
+            if Version(envd_version) <= Version("0.3.37"):
+                if src_path.is_dir():
+                    envd_config += (
+                        f'    io.copy(host_path="{src_path.as_posix()}", envd_path="/root/{src_path.as_posix()}/")\n'
+                    )
+                else:
+                    envd_config += (
```

Comment:
it should be `io.copy(host_path="...", envd_path="..."`), right

Reply:
Sorry for not noticing that. Will make a quick fix. Thanks a lot ~